Documentation/gettingstarted/clustermesh.rst

```diff
@@ -335,6 +335,34 @@ Cilium will automatically perform load-balancing to pods in both clusters.
      selector:
        name: rebel-base
 
+Load-balancing with Shared Services
+###################################
```

Comment:
Sorry, I just noticed this, but I think that this is more like what is achieved here:

```suggestion
Load-balancing only to a remote cluster
#######################################
```

Would you mind once more adjusting this? With this in place I think we're good to merge.

(In general it's not entirely obvious how this configuration achieves this but after thinking a bit I think I get it and this seems like a reasonable way to document it.)

Reply:
Done

Reply:
@chowmean would you mind also squashing? There's no need to spread these changes across multiple commits.

Reply:
My bad, done